src/main.ts

```diff
@@ -205,6 +206,11 @@ const command: Command = {
 			describe: 'Disable lazy widget loading detection',
 			type: 'boolean'
 		});
+
+		options('serviceWorker', {
```

Comment:
I removed the call to `options()` so this will now only be supported via `dojorc`. As a follow up to that, it doesn't seem like we have anywhere to document non command line argument configuration options. Would adding a section to the README work?